experimental/attestation_common/src/keying_material.rs

```diff
@@ -30,41 +33,31 @@ const ASSERTION_TYPE: &str = "amd_sev_snp_report";
 /// Assertion used for remote attestation.
 #[derive(Serialize, Deserialize)]
 pub struct Assertion {
-    attestation_info: AttestationInfo,
```

Comment:
Renamed to attestation_report, since that's what it is normally called. We still need to figure out what to do with the additional data, but that's for another PR.